pkg/gce-cloud-provider/compute/gce.go

```diff
@@ -66,6 +66,7 @@ type ConfigFile struct {
 type ConfigGlobal struct {
```

Comment:
are these structs defined elsehwere? They feel like structs that should be defined in the token library or something

Reply:
These structs follow the GCE configuration file format. It's defined in kubernetes/legacy-cloud-providers but I don't think it's worth adding a whole extra dependency for just a few fields we need in the conf file. This isn't limited to tokens; project ID comes from this conf too.

I do wonder why this format isn't in kubernetes/cloud-provider-gcp and whether this is gonna be a problem. Looking into it